Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/math-functions/abs.adoc

= ABS
:description: The ABS() function returns an absolute number (for example, the positive value of a number).
:page-topic-type: reference

The `ABS()` function returns an absolute number (for example, the positive value of a number). The data type of the returned value will depend on the data type of the value passed to the `ABS()` function.

== Syntax

> The syntax for the `ABS()`function is as follows:

Reviewer: ```suggestion
The syntax for the `ABS()`function is:
```

[source,sql]
----
ABS(x)
----

The `ABS()` function requires one argument:

* `x`: An expression that evaluates to a number.

[NOTE]
====
The *ABS()* function will return the negation of the negative numbers.
====

== Examples

=== Absolute value of a negative number

The following example demonstrates how the `ABS()` function can be used to obtain the absolute value of a negative number:

[source,sql]
----
SELECT ABS(-10.25);
----

This returns an absolute value of the passed argument:

[source,sql]
----
+--------+
| f      |
+--------+
| 10.25  |
+--------+
----

=== ABS() function with an expression

> The following example demonstrates how the `ABS()` function can be used with an expression to obtain the absolute value of the result:

Reviewer: ```suggestion
This example demonstrates how to use the `ABS()` function with an expression to obtain the absolute value of the result:
```
